search-ui/src/App.vue

```diff
@@ -94,15 +94,14 @@ const isJestRunning = computed((): boolean => {
 
 const systemMessage = computed((): string => {
   // if SYSTEM_MESSAGE does not exist this will return 'undefined'. Needs to be null or str
-  const systemMessage = sessionStorage.getItem('SYSTEM_MESSAGE')
-  if (systemMessage) return systemMessage
+  const systemMessage = getFeatureFlag('banner-text')
+  if (systemMessage?.trim()) return systemMessage?.trim()
   return null
 })
-const systemMessageType = computed((): string => {
-  // if SYSTEM_MESSAGE_TYPE does not exist this will return 'undefined'. Needs to be null or str
-  const systemMessageType = sessionStorage.getItem('SYSTEM_MESSAGE_TYPE')
-  if (systemMessageType) return systemMessageType
-  return null
+
+const systemMessageType= computed((): string => {
```

Comment:
since this isn't getting set by anything you can just remove it and do what I suggested above